internal/provisioner/kops_provisioner_installation_sli.go

```diff
@@ -58,11 +60,19 @@ func (provisioner *KopsProvisioner) makeSLIs(clusterInstallation *model.ClusterI
 
 func (provisioner *KopsProvisioner) createInstallationSLI(clusterInstallation *model.ClusterInstallation, k8sClient *k8s.KubeClient) error {
 	wait := 60
+	logger := provisioner.logger.WithFields(log.Fields{
```

Comment:
I think we can just pass logger from the calling function as an argument, as it is already initialized there and it is more consistent with the rest of the codebase.